app/src/main/java/com/github/polypoly/app/menu/shared_component/TrophiesView.kt

```diff
@@ -0,0 +1,132 @@
+package com.github.polypoly.app.menu.shared_component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.onGloballyPositioned
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.github.polypoly.app.game.user.Trophy
+import com.github.polypoly.app.game.user.User
+import com.github.polypoly.app.game.user.allTrophies
+import com.github.polypoly.app.ui.theme.advancedShadow
+import com.github.polypoly.app.util.toDp
+
+
+/**
+ * The default size of trophies
+ */
+const val TROPHY_SIZE = 50
+
+/**
+ * The default padding between trophies
+ */
+const val PADDING_TROPHIES = 10
+
+/**
+ * A trophy view that the player has won or not
+ * @param trophy the trophy to display
+ * @param won if the trophy is won by the player
+ * @param selected if the trophy is selected, i.e. is highlighted compared to a normal trophy
+ * @param onClick what happens when the user click on the trophy
+ * @param disable if the trophy is not clickable
+ */
+@Composable
+fun TrophyView(trophy: Trophy, won: Boolean, selected: Boolean = false,
+               onClick: () -> Unit = {}, disable: Boolean = false) {
+
+    val trophyColor: Color = if (won) MaterialTheme.colors.primary else
+        MaterialTheme.colors.onSecondary
+
+    Box(
+        modifier = Modifier
+            .advancedShadow(
+                alpha = if (selected && !disable) 0.3f else 0f,
+                color = MaterialTheme.colors.onSecondary,
+                shadowBlurRadius = PADDING_TROPHIES.dp,
+                cornersRadius = TROPHY_SIZE.dp)
+            .clip(CircleShape)
+            .background(
+                color = trophyColor)
+            .background(
+                color = Color.White.copy(alpha = if (selected) 0f else 0.6f))
```

Comment:
`Color.White` could be replaced with `MaterialTheme.colors.background` to improve its meaningfulness